service/src/main/resources/templates/index.html

```diff
@@ -110,7 +85,48 @@
     };
   }
 
-  window.onload = function() {
+    // Examples cache
+    const examplesCache = {};
+
+    // Custom plugin for logic that happens before a parameters element is created, hardcoded
+    // to work with the dataset example in the PUT dataset operation.
+    const LoadExternalValuePlugin = () => {
+      return {
+        wrapComponents: {
+          // wrapping https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/parameters/parameters.jsx
+          parameters: (Original, { React }) => (props) => {
+            // These hardcoded values should be replaced with code that determines the values
+            // using the user's current selection.
+            const contentType = 'application/json'
+            const activeExamplesKey = 'dataset'
```

Comment:
Will this replace every catalog dataset with the example?

Reply:
This means that it will only affect examples (which are part of a `requestBody` parameter) that are called `dataset`. So this makes the replacement more specific.

Also note that it will only affect examples that have this path _and_ are using `externalValue`. So any examples that use `value` won't be affected.